test/Factory/ContainerAbstractServiceFactoryTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DotTest\Session\Factory;
+
+use Dot\Session\Factory\ContainerAbstractServiceFactory;
+use Exception;
```

Comment:
```suggestion
use PHPUnit\Framework\MockObject\Exception;
```
Then move this line lower so that the `use`s are alphabetical.